.buildkite/scripts/packagecloud_upload.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+curl -u "${PACKAGECLOUD_API_KEY}:" \
+     -F "package[distro_version_id]=190" \ # ubuntu 18.04 LTS id
+     -F "package[package_file]=@"$(ls *.deb) \
+     https://packagecloud.io/api/v1/repos/helium/internal/packages.json
```

Comment:
Since etl is a public repo should we make these packages public?

Reply:
Probably at some point :)  For now I just wanted to get something going